azure-communication-ui/calling/src/main/java/com/azure/android/communication/ui/calling/models/CallCompositePushNotification.java

```diff
@@ -20,12 +20,12 @@ public class CallCompositePushNotificationInfo {
     private final String eventTypeResult;
 
     /**
-     * Create {@link CallCompositePushNotificationInfo}.
-     * @param notificationInfoMap Notification info map.
+     * Create {@link CallCompositePushNotification}.
+     * @param notification Notification info map.
```

Comment:
double notification

map format not clear, i.e. what is expectation of key values? any early validation?

Reply:
Hi Adam, the first notification is param name. second is intro text. 

When we receive push from firebase, it is map, we pass it to native SDK. It is is valid it returns valid 'type'. While working on implementation, I will check how native SDK respond to invalid or empty map. Most probably it should return empty or null values.

Reply:
It should still document the expected map format, or link to external documentation that explains it.

end of the day it's still a map and a user should know what is in that map. I.e. if they are writing tests around the feature or something they may choose to mock this.

Regarding param `notification` you can elaborate on it there, i.e.
Firebase cloud messaging notification map (for more information see: [LINK])

It is viable that a customer may not use firebase and another notification mechanism, but still may want to use this code, so providing more context/information is valuable to an end user.